src/components/layout/Nav.jsx

```diff
@@ -123,7 +123,7 @@ export default function Nav({
           setDonorPage={setDonorPage}
         />
       )}
-      {userProfile ? (
+      {(enableUserProfile && userProfile) ? (
```

Comment:
Is this check necessary @Petap0w ? I don't think it should be if the profile button has been removed entirely.